django-stubs/core/management/commands/flush.pyi

```diff
@@ -1,6 +1,10 @@
+from typing import Any
+
 from django.core.management.base import BaseCommand
 from django.core.management.color import Style
 
 class Command(BaseCommand):
-    stealth_options: tuple[str]
+    stealth_options: tuple[str, str, str]
```

Comment:
Let's remove this. `BaseCommand` already defines `stealth_options: tuple[str, ...]`

```suggestion
```

Reply:
I removed this from `inpsectdb.pyi` too for consistency